client.go

```diff
@@ -720,9 +720,13 @@ func (c *Client) SandboxStore() sandbox.Store {
 }
 
 // SandboxController returns the underlying sandbox controller client
-func (c *Client) SandboxController() sandbox.Controller {
-	if c.sandboxController != nil {
-		return c.sandboxController
+func (c *Client) SandboxController(name string) sandbox.Controller {
+	// default sandboxer is shim
+	if len(name) == 0 {
```

Comment:
Shall we also remove the set of a default sandboxer here? @fuweid

Reply:
Yes.